lxd/config/map.go

```diff
@@ -111,17 +119,21 @@ func (m *Map) Dump() map[string]interface{} {
 
 // GetRaw returns the value of the given key, which must be of type String.
 func (m *Map) GetRaw(name string) string {
-	key := m.schema.mustGetKey(name)
 	value, ok := m.values[name]
-	if !ok {
-		value = key.Default
+	if !strings.HasPrefix(name, "user.") {
```

Comment:
I'd probably revert the logic and do:
```
if shared.IsUserConfig(name) {
    return value
}
```

Then keep the rest of the logic as it is after that.